src/main/resources/application.properties

```diff
@@ -0,0 +1,48 @@
+spring.datasource.url=jdbc:h2:mem:music
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=${username}
+spring.datasource.password=${password}
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.mvc.throw-exception-if-no-handler-found=true
+spring.web.resources.add-mappings=false
+spring.resources.static-locations=classpath:/static
+
+spring.h2.console.enabled=true
+
+#http://localhost:8080/h2-console/
+
+support.email=USERNAME@gmail.com
+spring.mail.host=smtp.gmail.com
+spring.mail.port=465
+spring.mail.protocol=smtps
+spring.mail.username=simpleMail637@gmail.com
+spring.mail.password=unusualPassword
+spring.mail.properties.mail.transport.protocol=smtps
+spring.mail.properties.mail.smtps.auth=true
+spring.mail.properties.mail.smtps.starttls.enable=true
+spring.mail.properties.mail.smtps.timeout=8080
+
+#spring.security.oauth2.client.registration.google.scope=email,profile,openid
+#spring.security.oauth2.client.provider.google.token-uri=https://www.googleapis.com/oauth2/v4/token
+#spring.security.oauth2.client.provider.google.authorization-uri=https://accounts.google.com/o/oauth2/v2/auth
+#spring.security.oauth2.client.provider.google.user-info-uri=https://www.googleapis.com/oauth2/v3/userinfo
+#spring.security.oauth2.client.provider.google.jwk-set-uri=https://www.googleapis.com/oauth2/v3/certs
+#spring.security.oauth2.client.provider.google.user-name-attribute=profile
+
+#security.oauth2.client.clientId=${clientId}
+#security.oauth2.client.clientSecret=${clientSecret}
+#security.oauth2.client.accessTokenUri=https://www.googleapis.com/oauth2/v4/token
+#security.oauth2.client.userAuthorizationUri=https://accounts.google.com/o/oauth2/v2/auth
+#security.oauth2.client.clientAuthenticationScheme=form
+#security.oauth2.client.scope=openid,email,profile
+#security.oauth2.resource.userInfoUri=https://www.googleapis.com/oauth2/v3/userinfo
+#security.oauth2.resource.preferTokenInfo=true
+#spring.security.oauth2.client.registration.google.authorization-grant-type=client_credentials
+
+spring.security.oauth2.client.registration.google.client-name=Google
+spring.security.oauth2.client.registration.google.client-id=${clientId}
```

Comment:
If clientId is an Environment variable - please rename it, something like MUSIC_GOOGLE_CLIENT_ID would be better

Reply:
renamed in a different style